observability/WFLY-19846-promote-mp-tel-2-to-default.adoc

```diff
@@ -0,0 +1,96 @@
+---
+categories:
+- microprofile
+- observability
+stability-level: default
+issue: https://github.com/wildfly/wildfly-proposals/issues/619
+feature-team:
+ developer: jasondlee
+ sme:
+  - jasondlee
+ outside-perspective:
+  -
+promotes: https://issues.redhat.com/browse/WFLY-19590
+promoted-by: https://issues.redhat.com/browse/WFLY-19846
```

Comment:
These should be the link to the GitHub issue. The idea will be to have tooling to create a link between the two generated documents.

That said, I know the old issue, likely doesn't have a GitHub Issue associated with it. I'm not sure what we do in that case. However, of the `promoted-by` that should be a link to the GitHub Issue.

Reply:
Updated.